server/metrics.go

```diff
@@ -65,6 +65,14 @@ var (
 			Help:      "Status of the scheduler.",
 		}, []string{"kind", "type"})
 
+	balancerCounter = prometheus.NewCounterVec(
```

Comment:
Seems it collect metrics from other schedulers too, maybe rename to `schedulerCounter`?